gsy_e_sdk/setups/Asset_API_scripts/rest_bid_offer_posting.py

```diff
@@ -0,0 +1,147 @@
+# flake8: noqa
```

Comment:
Why are you disabling flake8?

Reply:
Please answer.

Reply:
Flake8 was disabled to disable the format test, now this line can be deleted.